### PR TITLE
add getNoticesByCategory endpoint

### PR DESCRIPTION
**Pull request by estasjuk**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -5,29 +5,29 @@ const logger = require("morgan");
 const cors = require("cors");
 const swaggerUi = require("swagger-ui-express");
 const swaggerDocument = require("./swagger.json");
-const path = require('path');
+// const path = require('path');
```
Comment (pzaloznyi) [inline app.js]: Please remove comments, the code should be clear. Or provide the reason why it should be commented.

File: app.js [hunk 2]
```diff
@@ -5,29 +5,29 @@ const logger = require("morgan");
...
 
 // const authRouter = require("./routes/api/auth/auth-routes");
 const noticeRouter = require("./routes/api/notices/notices-routes");
 // const userRouter = require("./routes/api/user/user-routes");
 
 const app = express();
-const router = express.Router();
+//  const router = express.Router();
```
Comment (pzaloznyi) [inline app.js]: also comments

File: app.js [hunk 3]
```diff
@@ -5,29 +5,29 @@ const logger = require("morgan");
...
 
-router.get('/', (req, res) => {
-  res.sendFile(path.join(__dirname, '/index.html'))
-});
+// router.get('/', (req, res) => {
+//   res.sendFile(path.join(__dirname, '/index.html'))
+// });
```
Comment (pzaloznyi) [inline app.js]: and here

File: app.js [hunk 4]
```diff
@@ -5,29 +5,29 @@ const logger = require("morgan");
...
 
 
 const formatsLogger = app.get("env") === "development" ? "dev" : "short";
 
-app.use('/', router);
+// app.use('/', router);
```
Comment (pzaloznyi) [inline app.js]: here

**Review by pzaloznyi — COMMENTED** [review]
please remove the commented lines above

File: app.js (hunk 3 above)
Comment (estasjuk) [reply app.js]: fixed

**Review by pzaloznyi — APPROVED** [review]
can be merged

**Review by pzaloznyi — CHANGES_REQUESTED** [review]
can wait till s3 implemented

**Pull request closed without merging**